protobuf/sliverpb/constants.go

```diff
@@ -280,6 +280,12 @@ const (
 	// MsgMv - Confirms the success/failure of the mv request (resp to MsgMvReq)
 	MsgMv
 
+	// MsgCpReq - Request to copy a file from one place to another
+	MsgCpReq
+	// MsgCp - Confirms the success/failure, as well as the total number of bytes
+	// written of the cp request (resp to MsgCpReq)
+	MsgCp
+
```

Comment:
Looks like [the warning wasn't big enough] (https://github.com/BishopFox/sliver/blob/master/protobuf/sliverpb/constants.go#L25-L38): please move these at the end, otherwise you'll break compatibility with other versions of the implant.